src/ArcGISRuntime.Samples.Shared/Models/SampleModel.cs

```diff
@@ -204,9 +222,15 @@ internal static SampleModel Create(string metadataFilePath)
                 sampleModel.SampleFolder = metadataFile.Directory;
             }
 
+            // Stop if the sample doesn't have a VB implementation (VB sample viewer only)
+            if (ApplicationManager.Current.SelectedLanguage == Language.VBNet && System.Type.GetType(sampleModel.SampleVbTypeName, false) == null)
```

Comment:
I've investigated this and it turns out the XAML files aren't included in the output directory, so that can't be used reliably to check that the sample was implemented. Because most of the properties in the sample model are taken from metadata.json (which is shared between the two types) I think reflection is the only way to go. :/